test/helpers/providers/nebius.bash

```diff
@@ -22,6 +22,7 @@ export GPU_OPERATOR_NAMESPACE="${GPU_OPERATOR_NAMESPACE:-gpu-operator}"
 export GPU_OPERATOR_VERSION="${GPU_OPERATOR_VERSION:-v25.3.4}"
 export GPU_OPERATOR_DRIVER_VERSION="${GPU_OPERATOR_DRIVER_VERSION:-570.195.03}"
 export NB_NODEGROUP_NAME="${NB_NODEGROUP_NAME:-github-ci-Nebius}"
+export NB_NODE_COUNT="${NB_NODE_COUNT:-1}"
```

Comment:
Node count is not multi-GPU. We should set this variable as a list of nodes:
1. 2 nodes of 1xGPU (so parallelism can be 2 for the tests)
2. 1 node of 8xGPU (run with parallelism 1)

Reply:
the node count was meant to increase the nodes for single gpu